storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/Tweet.java

```diff
@@ -14,6 +17,7 @@
 // Which will generate type mapping code in compile time,
 // You just need to link it in your code.
 @StorIOSQLiteType(table = TweetsTable.TABLE)
+@StorIOContentResolverType(uri = TweetMeta.URI_STRING)
```

Comment:
Yep, valid point :sweat_smile: We need to create clean separate examples of working with StorIO